bash2048.sh

```diff
@@ -289,15 +355,15 @@ function end_game {
     test "$REPLY" = "Y" || test "$REPLY" = "y" && {
       save_game
       printf "\nGame saved. Use -r option next to load this game.\n"
+      prompt_to_save_high_score $score $end_time
       exit 0
     }
     test "$REPLY" = "" && {
       printf "\nGame not saved.\n"
+      prompt_to_save_high_score $score $end_time
```

Comment:
this is another highscore saving function inside "savegame" section

Reply:
I am not sure I understand what you mean...

Reply:
same here, not sure to understand the link with savegame

Reply:
see the reply in conversation above